server/routes/ssot.js

```diff
@@ -9,8 +9,8 @@ const variableController = require('../controllers/ssotVariableController');
 const rpaAttributesController = require('../controllers/ssotRpaAttributes');
 
 /// PARSING ROUTES ///
-router.get('/parser/get-robot-code', parsingController.getRobotCode);
-router.get('/parser/getForId/:botId', parsingController.getRobotCodeForId);
+router.get('/parser/get-robot-code', parsingController.getRobotCode); // not used anymore
```

Comment:
why not delete all the routes that aren't used anymore?

Reply:
Because they are called at other places and this would be a refactoring that would expand over the user story time and bloat the pr.